index.html

```diff
@@ -274,6 +273,95 @@ <h2 class="contact-h2">Contact me</h2>
             </section>
         </section>
     </div>
+</div>
+      <div id="mobile-popup-container" class="container-modal display-none">
+    <div class="modal-head">
+        <div class="modal-h2-ul">
+            <h2 class="modal-h2">Tonic</h2>
+             <ul class="modal-first-ul">
+            <li class="modal-first-li mfl-canopy"></li>
+            <li class="modal-first-li mfl-dev"></li>
+            <li class="modal-first-li mfl-year"></li>
+        </ul>
+        </div>
+        
+        <div class="modal-close-button">&times;</div>
+    </div>
+    <div class="modal-img-container">
+    </div>
+    <div class="modal-mid-sec">
+        <p class="modal-sec-p">
+            Lorem Ipsum is simply dummy text of the printing and 
+            typesetting industry. Lorem Ipsum has been the industry's 
+            standard dummy text ever since the 1500s, when an unknown printer 
+            took a galley of type and scrambled it to make a type specimen book. 
+            It has survived not only five centuries, but also the leap into
+             electronic typesetting, remaining essent
+        </p>
+        <div class="modal-sec-ul-button">
+            <ul class="modal-sec-ul">
+                <li class="modal-sec-li mfl-html">HTML</li>
+                <li class="modal-sec-li mfl-css">CSS</li>
+                <li class="modal-sec-li mfl-js">Javascript</li>
+            </ul>
+            <div class="modal-button">
+                <button class="modal-btn-seelive" type="button">See Lives <img src="images/seeLiveIcon.png" alt="see live button"></button>
+                <button class="modal-btn-seesource" type="button">See Source<img src="images/seeSource.png" alt="see live button"></button>
+                </div>
+        </div>
+        
+    </div>
+        
+</div>
+<!-- desktop popup -->
+<div id="desktop-popup-container" class="container-modal display-none">
```

Comment:
- [ ] Please make sure the popup is displayed as in the design, the page content should not be  visible when the popup is displaying, see image below and kindly fix that.

***Your implementation***
![grace01](https://user-images.githubusercontent.com/65852621/137262008-18447868-2339-4d40-b0f1-eafab7658d20.png)

***The design***
![graceD01](https://user-images.githubusercontent.com/65852621/137262117-c0082973-7b4e-42e2-a4b9-44cb89bab2a9.png)